joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/testfixtures/RubyCfgTestCpg.scala

```diff
@@ -0,0 +1,8 @@
+package io.joern.rubysrc2cpg.testfixtures
+
+import io.joern.x2cpg.testfixtures.CfgTestCpg
+
+class RubyCfgTestCpg extends CfgTestCpg with RubyFrontend {
```

Comment:
Surely we can use the existing `RubyCode2CpgFixture` class instead of create a new one?

Reply:
Addressed here: https://github.com/joernio/joern/pull/3069/commits/064e533e9025dd94b99dd397bdba7770a9f677eb